### PR TITLE
Project implementation

### PR DESCRIPTION
**Pull request by Lino09**
### In this PR I implement my project which is a weather app that can display the user's weather and location

in this PR I implemented the following:

- Setup Redux and React Router
- Addition of Axios and Thunk
- Implementation of the Home Page
- Implementation of Details Page
- Added a filter to see cities by their weather
- Unitary and Rederization Tests
- A live demo [Here](https://abels-weather-app.herokuapp.com/)
- A video to display the app functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+# Weather App - Capstone Project
```
Comment (TedLivist) [inline README.md:1]: - [x] In your README, kindly add an **Acknowledgements** section or similar that gives appropriate credit to Nelson Sakwa, the author of the original design, as required by the Creative Commons License.

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  };
+  const [localWeather, setWeather] = useState('');
+  const [localTemp, setTemp] = useState('');
+  const [localLocation, setLocalLocation] = useState('Microverse');
+  const [localCoords, setLocalCoords] = useState('');
+  const handleLocalWeather = () => {
+    if ('geolocation' in navigator) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        console.log('Latitude is :', position.coords.latitude);
+        console.log('Longitude is :', position.coords.longitude);
```
Comment (TedLivist) [inline src/components/Home.js]: - [x] Let's clean up your code a little bit. As stated in the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), it is not good practice to commit `console.log()` as it’s ugly, kills performance and can make confidential data visible to anyone. Kindly remove these console logs

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Lino09,

Good job so far!
✔️ Great video presentation
✔️ Kudos with the location extraction concept
✔️ Well-written tests
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Lino09) [reply README.md:1]: Thank! I can't Believe I forgot to do it

Comment (DeliceLydia) [reply README.md:1]: Well resolved this!

File: src/components/Home.js (hunk 2 above)
Comment (DeliceLydia) [reply src/components/Home.js]: well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Lino09, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Have Fun!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**